Code/Tools/Libs/GuiFoundation/NodeEditor/Implementation/Connection.cpp

```diff
@@ -168,4 +186,102 @@ void ezQtConnection::paint(QPainter* painter, const QStyleOptionGraphicsItem* op
   }
 
   painter->drawPath(path());
+
+  if (decorationFlags.IsSet(ezQtNodeScene::ConnectionDecorationFlags::DrawDebugging))
```

Comment:
There also needs to be something on the connection it self to tell whether it should show the debugging effect.

Reply:
Yea thats the intention at the moment. I haven't added any debug logic myself yet but it made sense to include the visuals in the changes i was making